README.md

```diff
@@ -148,6 +148,20 @@ Google is providing this collection of pre-implemented Dataproc templates as a r
           HISTORY_SERVER_CLUSTER=${HISTORY_SERVER_CLUSTER} \ #  [Optional]
           TEMPLATE_NAME=GCSTOBIGQUERY
         ```
+## Executing templates in existing dataproc cluster
+   
+To run the templates against existing cluster specify **JOB_TYPE** as dataporc and provide the **CLUSTER**. Following is an example command to run HIVETOBIGQUERY in existing dataproc cluster. 
+
+    ```
+    bin/start.sh GCP_PROJECT=${PROJECT} \
+    REGION=${REGION}  \
+    JOB_TYPE=dataproc \
+    CLUSTER=${DATA_PROC_CLUSTER_NAME}   \  # Specify name of existing dataproc cluster
+    GCS_STAGING_BUCKET=${GCS_STAGING_BUCKET} \
+    TEMPLATE_NAME=GCSTOBIGQUERY   \
```

Comment:
On line 153 it says **HIVETOBIGQUERY** but here it is calling the **GCSTOBIGQUERY** template